sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/UpdateCenterDeserializer.java

```diff
@@ -393,6 +397,13 @@ private void parseSonarLtsVersion(Properties p, Sonar sonar) {
     }
   }
 
+  private void parseLtaVersions(Properties properties, Sonar sonar) {
+    String ltaVersion = get(properties, LTA_VERSION, true);
+    String pastLtaVersion = get(properties, PAST_LTA_VERSION, true);
+    sonar.setLtaVersion(ltaVersion);
+    sonar.setPastLtaVersion(pastLtaVersion);
```

Comment:
I think we should keep the check on `sonar.getReleases().contains(sonar.getLtaRelease())` I feel it is useful if there is a typo in the field ltaVersion or pastLtaVersion, it would fail fast

Reply:
I was wondering what was the purpose of that check

Reply:
basically, you need to describe each version in the properties file. eg.  here for the [LTS](https://github.com/SonarSource/sonar-update-center-properties/blob/master/update-center-source.properties#L100)